benchexec/tools/relay-sv.py

```diff
@@ -0,0 +1,39 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for RELAY adapted for BenchExec.
+    URL: https://github.com/vesalvojdani/relay-sv
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("run_relay.sh")
+
+    def version(self, executable):
+        return self._version_from_tool(executable, line_prefix="Version: ")
+
+    def name(self):
+        return "relay-sv"
+
+    def cmdline(self, executable, options, task, rlimits):
+        return [executable, *options, *task.input_files]
```

Comment:
Does the tool support more than one input file? If not, please use `task.single_input_file`.

Reply:
The tool does, but the preprocessing has to be done manually in some complicated ways, so for now, the wrapper will only support a single file. I will fix this and the other comments. I'll do a test run on my machine, and then push it here. Thanks!